### PR TITLE
Add involved object namespace filtering

### PR DESCRIPTION
**Pull request by dgrisonnet**

### CODE REVIEW

File: internal/options/options.go [hunk 1]
```diff
@@ -64,6 +67,7 @@ func (o *Options) AddFlags() {
 	o.flags.StringVar(&o.ExporterHost, "exporter-host", "0.0.0.0", "Host to expose kube-events-exporter own metrics on.")
 	o.flags.IntVar(&o.ExporterPort, "exporter-port", 8081, "Port to expose kube-events-exporter own metrics on.")
 	o.flags.BoolVar(&o.Version, "version", false, "kube-events-exporter version information")
+	o.flags.StringArrayVar(&o.InvolvedObjectNamespaces, "involved_object_namespaces", []string{metav1.NamespaceAll}, "Events involved object namespace filter.")
```
Comment (lilic) [inline internal/options/options.go:70]: Can we write in the arg help text that it defaults to all namespaces

File: internal/collector/event.go [hunk 2]
```diff
@@ -115,3 +151,9 @@ func updatedEventNb(oldEv, newEv *v1.Event) int32 {
 
 	return newEv.Count - oldEv.Count
 }
+
+func filterInvolvedObjectNs(list *metav1.ListOptions, ns string) {
+	if ns != "" {
```
Comment (lilic) [inline internal/collector/event.go:156]: What if it is empty string? Something seems to have gone wrong then we want the user to know maybe with info log or debug log?

Comment (dgrisonnet) [reply internal/collector/event.go:156]: Empty string is the underlying value of `metav1.NamespaceAll`. I should definitely make it clearer in the condition.

**Comment by dgrisonnet** [comment]
@lilic I have addressed your comments, please have another look :slightly_smiling_face:

**Pull request merged**